docs/manual/extensions/cca-merger2.en.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Merger²"
+description: "Merger² is a frontend module to display modules/contents under certain conditions."
+url: "extension/merger2"
+---
+
+**[contao-community-alliance/merger2](https://packagist.org/packages/contao-community-alliance/merger2)**
+
+_von [Contao Community Alliance (CCA)](https://c-c-a.org/)_
+
+
+
+The Merger² extension for the CMS Contao offers a powerful frontend (FE) module to add content under certain conditions.
+Display or merge conditions. The contents can be articles, pages or other FE modules. The Conditions for the display
+can be set up in rules of any complexity.
+
+Examples for the versatile use are:
+
+* **Merge content** e.g. to display language-specific FE modules only if the corresponding language is
+  is called in the browser - thus several FE modules can be combined to one module and used in the layout;
+  this saves having to create several language-specific layouts.
+* Modules or articles are **only displayed if certain conditions are present**, e.g. a defined depth in the
+  Page tree, in the mobile display, when a certain language is called in the browser.
+* **Inheritance of articles** in the page tree, e.g. to automatically send an article from a superior page to all
+  Passing on children's pages
```

Comment:
"superior" -> "parent"

Aber der Satz ist generell komisch: "send an article from a parent page to all passing on children's pages"? Was ist hier gemeint?